apps/v1/types.go

```diff
@@ -14,6 +14,9 @@ import (
 // +genclient:method=GetScale,verb=get,subresource=scale,result=k8s.io/api/extensions/v1beta1.Scale
 // +genclient:method=UpdateScale,verb=update,subresource=scale,input=k8s.io/api/extensions/v1beta1.Scale,result=k8s.io/api/extensions/v1beta1.Scale
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=3.0
+// +k8s:prerelease-lifecycle-gen:deprecated=4.14
+// +k8s:prerelease-lifecycle-gen:removed=4.10000
```

Comment:
do we need to put a removal target in at all?  i'd prefer not to even imply it's going to be removed until/unless we update our api compatibility guidelines.

Reply:
Unfortunately we do. Otherwise, it defaults to deprecated+3.